### ISSUE TITLE
'punycode' error

### ISSUE THREAD

**Issue by Xheldon** [ProseMirror/website#111]
I'm using node version v10.

After i run `make` cmd and when i open the /examples/markdown/ website path, i got an error show that `punycode` is not defined.

I checked it and find this error in make cmd:
```bash
node bin/build-example.js example/markdown/index.js > public/examples/markdown/example.js
'punycode' is imported by ../node_modules/markdown-it/lib/index.js, but could not be resolved – treating it as an external dependency
'punycode' is imported by commonjs-external:punycode, but could not be resolved – treating it as an external dependency
Creating a browser bundle that depends on Node.js built-in module ('punycode'). You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
No name was provided for external module 'punycode' in options.globals – guessing 'punycode'

```

Also i find this: https://github.com/markdown-it/markdown-it/issues/230#issuecomment-334914902

So I know the reason is punycode is node build-in module, and should use some special way to build it. But I find that your website is work well in markdown example: https://prosemirror.net/examples/markdown/  but not mine: https://prosemirror.xheldon.com/examples/markdown/

So I want to know What the magic do you use? Thanks!

**Comment by Xheldon** [comment]
Besides, I use this CI to build: https://travis-ci.org/github/xheldon-prosemirror/prosemirror/builds/712056852

It's simply run `yarn` & `bin/pm install` then push it to remote github pages repo

**Comment by Xheldon** [comment]
I find the resolution : #112    @marijnh

**Comment by marijnh** [comment]
On my machine, using node 14, the build process doesn't have any problems. `punycode` is in the website's own package.json, so Rollup can resolve it just fine. Is it present in your node_modules dir?

**Comment by Xheldon** [comment]
> On my machine, using node 14, the build process doesn't have any problems. `punycode` is in the website's own package.json, so Rollup can resolve it just fine. Is it present in your node_modules dir?

I try node 14 with nvm, and the problem is still there. And the package `punycode` is always in the node_modules dir no matter which version of node i used, i try to clean the cache of npm(just make sure), nothing changes.

Besides, I `mkdir` a new folder to git clone this repo, and run `npm i`(or `yarn`, both i have tried with different directory---another make sure) use node v14, then run `make`, but the error always appear.

Seriously, I don't want waste your time with this problem, and I know it's maybe just a environment or package version of rollup&markdown-it or something else problem, and i fix this problem with this pr said:  #112  , but i still want to ask...:
Have you try to rerun git clone this repo and run the `npm i`  & `make` command in a new directory?
If you insist there is no problem in local machine, just close this issue😂
Thanks!

**Comment by marijnh** [comment]
Ah, good point, when reinstalling from scratch I do get the error. It appears to be caused by a regression in rollup-plugin-node-resolve. I've upgraded the repository to use `@rollup/plugin-node-resolve` instead in patch 44cac70947ded9, which seems to fix the problem.
